solana/programs/ntt-quoter/src/state.rs

```diff
@@ -0,0 +1,46 @@
+use anchor_lang::prelude::*;
+
+#[account]
+#[derive(Debug, InitSpace)]
+pub struct Instance {
+    pub owner: Pubkey,
+    pub assistant: Pubkey,
+    pub fee_recipient: Pubkey,
+    pub sol_price: u64, //UsdPrice (usd/sol [6 decimals])
+                        //amount of sol the user has to pay for the relayer to trigger the release of their outbox message
+                        //pub release_cost: u64, //SolAmount (lamports)
+}
+
+impl Instance {
+    pub const SEED_PREFIX: &'static [u8] = b"instance";
+    pub const BUMP: u8 = crate::INSTANCE_BUMP;
```

Comment:
Yeah, it's an option to rely on the account discriminator alone, rather than also check the PDA. I didn't spend a lot of time worrying about future compatibility - I think the only truly relevant issue in this regard is making sure that the program account can be upgraded to a larger program if there ever is the need for a more powerful/scalable solution.

This is very much an quick mvp.